### ISSUE TITLE
Add Promise polyfill

### ISSUE THREAD

**Issue by tarun** [tschaub/karma-phantomjs-shim#4]
Only IE & PhantomJS do not have a native Promise implementation now.
Would it be appropriate to polyfill promise in this shim?

Encountering this issue mainly with `axios` lib which requires a Promise polyfill to work.
And it is a pain to load the polyfill in karma before the test scrips.

**Comment by tschaub** [comment]
Yes, this would be nice to add.  Rather than including the source in this repo, it would make sense to add [`lie`](https://www.npmjs.com/package/lie) or another implementation as a dependency and then have a build step to generate the shim for distribution.
